.github/workflows/draft-release.yml

```
@@ -0,0 +1,36 @@
+name: 🏗️ Draft New Release
+
+on:
+  push:
+    tags:
+      - '^[0-9]+\.[0-9]+\.[0-9]+$'
```

actionlint

character '^' is invalid for branch and tag names. ref name cannot contain spaces, ~, ^, :, [, ?, *. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)